examples/advanced/tidb-cluster.yaml

```diff
@@ -1,295 +1,872 @@
     # affinity:
     #   podAntiAffinity:
+    #     # prefer not to run pd pods on the same node which runs tidb/tikv pods
     #     preferredDuringSchedulingIgnoredDuringExecution:
-    #       - podAffinityTerm:
-    #           labelSelector:
-    #             matchExpressions:
-    #               - key: app.kubernetes.io/component
-    #                 operator: In
-    #                 values:
-    #                   - tidb
-    #                   - tikv
-    #           topologyKey: ""
-    #         weight: 100
-    ## If you use require affinity, you must ensure that at least 3 nodes are available in the cluster
-    #     requiredDuringSchedulingIgnoredDuringExecution:
-    #       - labelSelector:
+    #     - podAffinityTerm:
+    #         labelSelector:
     #           matchExpressions:
-    #             - key: app.kubernetes.io/component
-    #               operator: In
-    #               values:
-    #                 - pd
-    #         topologyKey: ""
-    
-    ## Specify node labels for pods assignment
-    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
-    # nodeSelector:
-    #   app.kubernetes.io/component: pd
+    #           - key: app.kubernetes.io/component
+    #             operator: In
+    #             values:
+    #             - tidb
+    #             - tikv
+    #         topologyKey: kubernetes.io/hostname
+    #       weight: 100
+    #     # require not to run PD pods on nodes where there's already a PD pod running
+    #     # if setting this, you must ensure that at least `replicas` nodes are available in the cluster
+    #     requiredDuringSchedulingIgnoredDuringExecution:
+    #     - labelSelector:
+    #         matchExpressions:
+    #         - key: app.kubernetes.io/component
+    #           operator: In
+    #           values:
+    #           - pd
+    #       topologyKey: kubernetes.io/hostname
+
+    ## set a different tidb client TLS cert secret name for TiDB Dashboard than the default ${clusterName}-tidb-client-secret
+    ## only useful when TLS is enabled for TiDB server
+    ## Ref: https://docs.pingcap.com/tidb-in-kubernetes/stable/enable-tls-for-mysql-client
+    # tlsClientSecretName: custom-tidb-client-secret-name
 
   tidb:
-    # Base image of the component
-    baseImage: pingcap/tidb
+    ############################
+    # Basic TiDB Configuration #
+    ############################
 
-    # The desired ready replicas
-    replicas: 3
+    ## Base image of the component
+    baseImage: pingcap/tidb
 
-    # describes the compute resource requirements.
-    # Ref: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
-    requests:
-      cpu: "100m"
+    ## tidb-server Configuration
+    ## Ref: https://docs.pingcap.com/tidb/stable/tidb-configuration-file
+    config: |
+      mem-quota-query = 536870912
+      token-limit = 100
```

Comment:
Are these values the recommended ones? If not sure, what about just put a log level config here?